src/resourceGenUtils.js

```diff
@@ -48,7 +49,8 @@ const scriptsToAdd = {
   'template-init': 'sgtu-init',
   test: 'echo "test"',
   'test-connection': 'echo "test-connection"',
-  'test-to-url': 'echo "test-to-url"',
+  'test-to-url': 'istanbul cover ./node_modules/mocha/bin/_mocha ' +
```

Comment:
please update the "test" script to run transform/toUrl/connection tests and the istanbul stuff... I think the individual "test-transform" and "test-to-url" and "test-connection" should just run the tests and NOT do the coveralls stuff
what do you think?